backend/unitTests/addItemTest.py

```
@@ -0,0 +1,103 @@
+    
+    return f"data:image/png;base64,{image_data}"
+
+def test_add_basic_item():
+    """Test adding a basic DIY item"""
+    item_data = {
+        "username": "test_user",  # SECURITY VULNERABILITY - No authentication
+        "image": create_test_image_data()
+    }
+    
+    try:
+        response = requests.post(f"{BASE_URL}/api/items/add", json=item_data)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)